pages/common/airodump-ng.md

```diff
@@ -0,0 +1,13 @@
+# airodump-ng
+
+> Capture packets and display information about wireless networks.
+> Part of `aircrack-ng`.
+> More information: <https://www.aircrack-ng.org/doku.php?id=airodump-ng>.
+
+- Capture packets and display information about a wireless network:
+
+`sudo airodump-ng {{interface}}`
+
+- Capture packets and display information about a wireless network, saving the data to a file:
+
+`sudo airodump-ng --channel {{channel}} --write {{filename}} --bssid {{mac}} {{interface}}`
```

Comment:
From the differences of the descriptions I would expect the commands to only differ by the `--write` flag. What does `--channel` and `--bssid` do and shouldn't they be introduced by a separate example?